schedule/schedule-appengine/src/main/client/jasify/services/multipass.service.js

```diff
@@ -3,86 +3,41 @@
 
     function multipass(Endpoint) {
```

Comment:
You could try to move this service to use the generated jasify client.  Would be an excellent proof of concept, and then you can tell me if it is simpler or not.